### PR TITLE
Feature/nfs module

### PR DESCRIPTION
**Pull request by jadnov**
nfs module, standalone and integrated in slurm template

### CODE REVIEW

File: slurm/versions.tf [hunk 1]
```diff
@@ -7,5 +7,5 @@ terraform {
 }
 
 provider "nebius" {
-  domain = "api.eu.nebius.cloud:443"
+  domain = "api.eu-north1.nebius.cloud:443"
```
Comment (asteny) [inline slurm/versions.tf:10]: ?

File: slurm/files/ansible_role.zip
Comment (d3vil-st) [inline slurm/files/ansible_role.zip:1]: I would suggest to remove that file and add into .gitignore

File: slurm/versions.tf (hunk 1 above)
Comment (jadnov) [reply slurm/versions.tf:10]: mistake, fixing

File: slurm/files/ansible_role.zip
Comment (jadnov) [reply slurm/files/ansible_role.zip:1]: Ilya, you forget this is actually an ansible file transferred to VM, without it will not work

**Review by jadnov — COMMENTED** [review]
ok

**Pull request merged**